packages/radix-ui-themes/src/components/segmented-control.css

```
@@ -20,6 +20,12 @@
 
   /* Create a new stacking context */
   isolation: isolate;
+
+  &.disabled {
```

Should we use `:where` or is the added specificity useful here?
```suggestion
  &:where(.disabled) {
```

---

The specificity is needed here to override the `color` and `background-color` on the base class.